### ISSUE TITLE
version mismatch of binary files in conda python libraries

### ISSUE THREAD

**Issue by lrlunin** [slsdetectorgroup/slsDetectorPackage#679]
##### *Distribution: 

RHEL7

##### *Detector type: 

moench

##### *Software Package Version: 

6.1.2, 2022.2.3.dev0, 7.0.0
##### Priority:

SUPER HIGH
##### *Describe the bug

we have a virtual detector built from source from the tag `2022.2.3.dev0`. I tried to migrate to your distribution of `slsdet` libraries.

If I install the libraries:
```yaml
    - slsdetlib=2022.2.3.dev0
    - slsdet=2022.2.3.dev0
```
then I start the `slsReceiver` given in the path and then I try to use `sls_detector_put config sample.config` I obtain the following error message:
```
- 18:01:04.051 ERROR: Detector shared memory (0) version mismatch (expected 0x201007 but got 0x220505. Clear Shared memory to continue.
- 18:01:04.051 ERROR: Shared memory version mismatch!
```
When I use my own slsReceiver/sls_detector_put binaries built from the tag `2022.2.3.dev0` together with the virtual detector there is no issue.

##### Expected behavior

updated libraries 
##### To Reproduce

install from conda distribution
```yaml
    - slsdetlib=2022.2.3.dev0
    - slsdet=2022.2.3.dev0
```
start `slsReceiver` from the conda, a virtual detector built from source from tag  `2022.2.3.dev0` and try to upload the config with `sls_detector_put config` from conda binaries

**Comment by lrlunin** [comment]
any other newer version have the same bug

**Comment by lrlunin** [comment]
I didn't test it with the real detector but I guess the problem will be the same. Self compiled software works fine with both virtual and real detector.

**Comment by lrlunin** [comment]
I've just tested it with the from the sources compiled `slsReceiver` and `sls_detector_put config` with the virtual detector and it works just fine. The issue is wrong version of `slsReceiver` and other executables.

**Comment by erikfrojdh** [comment]
I failed to reproduce this issue:

**Download and build the virutal server from tag: 2022.2.3.dev0**
```bash
git clone git@github.com:slsdetectorgroup/slsDetectorPackage.git --branch=2022.2.3.dev0 --recursive
mkdir build && cd build
cmake ../slsDetectorPackage/ -DSLS_USE_SIMULATOR=ON
make -j12

#Launch virtual server
bin/moenchDetectorServer_virtual
```

**Install slsdet from conda**
```bash
conda create -n versiontest slsdet=2022.2.3.dev0 #new env with the tag
conda activate versiontest
sls_detector_put config minimal.config

#Read back versions
>> sls_detector_get versions
versions 
Detector Type: Moench
Package Version: developer
Client Version: 0x211125
Firmware Version: 0x0
Detector Server Version: 0x220202
Detector Server Version: #1 SMP PREEMPT_DYNAMIC Thu Feb  9 19:20:24 UTC 2023
Receiver Version: 0x211124

```

Could it be that you had already the shared memory configured with the old software?

**Comment by lrlunin** [comment]
Thank you so much for your help, @erikfrojdh . Without you I would have never found the problem. The problem was the forked and then wrongly merged, don't know why but it was merged with the 6.1.2 version and then everything was broken.

I am really sorry for disturbance and thank you so much for the help. I owe you a lot ;-)
